docs/docs/topics/bus-event-handling.mdx

```diff
@@ -7,7 +7,7 @@ title: Bus Event handling
 :::warning Under Construction
 
 This page is still under construction and is not available yet.<br />
-Please reach out in Slack if you have some questions about the **Bus events**
+Please reach out on Discord if you have some questions about the **Bus events**
```

Comment:
I know this is just a fix from what we had previously but I don't know if we need to specifically mention in certain areas that a we can respond to questions in the chat. But perhaps it's since we have the under construction part here as well as in the other location where we previously mentioned slack.

Reply:
Sure. I just meant "reach out on Discord" should apply for any question throughout the docs, not specifically on certain pages.